buffer/src/main/java/io/netty5/buffer/Buffer.java

```diff
@@ -357,6 +357,23 @@ default int writableBytes() {
      */
     int transferTo(WritableByteChannel channel, int length) throws IOException;
 
+    /**
+     * Read from this buffer and write to the given channel at the given position.
+     * The number of bytes actually written to the channel are returned.
+     * No more than the given {@code length} of bytes, or the number of {@linkplain #readableBytes() readable bytes},
+     * will be written to the channel, whichever is smaller.
+     * The channel's position is not modified.
+     * The {@linkplain #readerOffset() reader-offset} of this buffer will likewise be advanced by the number of bytes
+     * written.
```

Comment:
```suggestion
     * The {@linkplain #readerOffset() reader-offset} of this buffer will, however, be advanced by the number of bytes
     * written.
```

Reply:
i'll change it manually and in the old transferTo too

Reply:
ahh i see.